extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -278,6 +295,16 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * Receives any response that may be sent from the cross-domain iframe
+   * @param {String} msg The response message from the iframe
+   * was specified in the amp-analytics config
+   */
+  processCrossDomainIframeResponse_(msg) {
+    // Used in ../../../src/anchor-click-interceptor.js
+    this.win.document.mostRecentCrossDomainIframeResponse = msg;
```

Comment:
True, but enclosed in what? AnchorClickInterceptor knows what "A" tag has been clicked, and can get from there to the enclosing document in O(1) time. But if I store it in the amp-analytics tag then we'd have to go walk the DOM tree to find it at the time of the click.